utils/zerovec/src/map/borrowed.rs

```diff
@@ -201,57 +201,42 @@ impl<'a, K, V> ZeroMapBorrowed<'a, K, V>
 where
     K: ZeroMapKV<'a> + ?Sized,
     V: ZeroMapKV<'a, Container = ZeroVec<'a, V>> + ?Sized,
-    V: AsULE + Ord + Copy,
+    V: AsULE + Ord + Copy + 'static,
 {
     /// For cases when `V` is fixed-size, obtain a direct copy of `V` instead of `V::ULE`
     pub fn get_copied(&self, key: &K) -> Option<V> {
         let index = self.keys.binary_search(key).ok()?;
-        <[V::ULE]>::get(self.values, index)
-            .copied()
-            .map(V::from_unaligned)
+        self.values.get(index)
     }
 
     /// Similar to [`Self::iter()`] except it returns a direct copy of the values instead of references
     /// to `V::ULE`, in cases when `V` is fixed-size
     pub fn iter_copied_values<'b>(
         &'b self,
     ) -> impl Iterator<Item = (&'b <K as ZeroMapKV<'a>>::GetType, V)> {
-        (0..self.keys.len()).map(move |idx| {
-            (
-                self.keys.get(idx).unwrap(),
-                <[V::ULE]>::get(self.values, idx)
-                    .copied()
-                    .map(V::from_unaligned)
-                    .unwrap(),
-            )
-        })
+        (0..self.keys.len())
+            .map(move |idx| (self.keys.get(idx).unwrap(), self.values.get(idx).unwrap()))
```

Comment:
Suggestion, here and elsewhere: Can we use `Iterator::zip` to avoid the `unwrap` ?  (this is pre-existing)

Reply:
Maps don't (and can't) have access to iterators because we can't talk about `impl Trait` in trait impls (_yet_, impl-trait-in-type-alias fixes this). We can achieve this by using named iterator types in ZeroVec and VarZeroVec but that's a lot more boilerplate.